lib/cronofy/client.rb

```diff
@@ -861,7 +861,7 @@ def availability(options = {})
     #                                of minutes for the availability query.
     #             :buffer            - An Hash containing the buffer to apply to
     #                                the availability query.
-    #           :available_periods - An Array of available time periods Hashes,
+    #             :available_periods - An Array of available time periods Hashes,
```

Comment:
Should this change too?

Reply:
Good spot, initially I didn't think so but we did make the same change here. I've updated the PR to do the same backwards-compatible reading of `query_periods` here